src/DynamoCoreWpf/ViewModels/Watch3D/Shaders/Custom.fx

```diff
@@ -103,14 +103,7 @@ float4 PShaderCustom(PSInputCustom input) : SV_Target
 	}
 
 	/// set diffuse alpha
-	if (!isIsolationMode || isSelected)
-	{
-		I.a = vMaterialDiffuse.a;
-	}
-	else
-	{
-		I.a = vTransparentMaterial.a;
-	}
+	I.a = vMaterialDiffuse.a;
```

Comment:
@Benglin Reverting this one back to the implementation before Isolate Selected Geometry, as the transparency calculation is to be done after the assignment of `I`.